unidist/cli/base/utils.py

```diff
@@ -0,0 +1,121 @@
+# Copyright (C) 2021 Modin authors
+#
+# SPDX-License-Identifier: Apache-2.0
+
+"""Utilities for backend runners."""
+
+import os
+import ipaddress
+import socket
+import multiprocessing as mp
+
+
+class BackendName:
```

Comment:
I think we can move this to `unidist/core/base/common.py` to use it where possible instead of using string values as we do now.

Reply:
Not sure that I understand your idea, can you give more description?

Reply:
Move `BackendName` in `unidist/core/base/common.py` so we can use it in other places, but not string values like "Mpi", "Ray", etc.

Reply:
note: direct using `BackendName` in other places should happen in a separate PR.

Reply:
Oh, I see, agree. Thanks for explanation. It will be done as resolution of #47

Reply:
We can/should move `BackendName` in `unidist/core/base/common.py` as part of this PR and start using it in other places as part of another one.

Reply:
`BackendName` was moved in `unidist/core/base/common.py` via 49df1c37e9f879e2dafa6d9fe779743c94dc79ea